tests/unit-tests/data-port/models/test-class-sensei-data-port-question-model.php

```diff
@@ -345,7 +345,7 @@ public function testValidGapFillPostCreated() {
 		$this->assertTrue( $result );
 
 		$post = get_post( $model->get_post_id() );
-		$this->commit_transaction();
```

Comment:
Is this here by accident?

Reply:
I think it _was_ there by accident, which is why I removed it 😃